internal/pkg/describe/env.go

```diff
@@ -128,6 +160,15 @@ func (e *EnvDescriber) getStackName(resourceArn string) (string, error) {
 	return stack[1], nil
 }
 
+func (e *EnvDescriber) envOutputs() ([]*CfnResource, error) {
+	envStack, err := e.stackDescriber.StackResources(stack.NameForEnv(e.app.Name, e.env.Name))
+	if err != nil && !IsStackNotExistsErr(err) {
+		return nil, fmt.Errorf("retrieve environment resources: %w", err)
+	}
+	outputs := flattenResources(envStack)
+	return outputs, nil
```

Comment:
What happens if the err is a stack not exists err?

Reply:
Ah, good point. I'm seeing, in other usages, that if it IS an IsStackNotExistsErr then "continue." But because this isn't a loop, do I need to wrap the IsStackNotExistsErr? Or just return err?

Reply:
What do you think of just catching any error here and returning that? I feel like as a customer, I wouldn't be surprised to receive an error if the environment stack doesn't exist and I call "env show" on it